defaults/main.yml

```diff
@@ -91,6 +91,10 @@ jenkins_root: "/opt/jenkins"
 # Jenkins user.
 jenkins_secret_files: []
 
+# If true, show the diff output when copying the plugins.yaml file to the controller. This
+# can be useful to see which plugins during a deployment.
```

Comment:
@nre-ableton there seems be a word missing here.

Reply:
Addressed in https://github.com/Ableton/ansible-role-jenkins-jcasc/pull/262.